Bikeini/screens/BikeInformation.js

```diff
@@ -220,24 +227,45 @@ class BikeInformation extends React.Component {
       const {
         author,
       } = item;
+      const { _id } = item;
       const { jwt } = authState;
       const ownersComment = profileState.username === item.author.username;
       bikeData.showResolveBike = bikeData.submitter.username !== item.author.username && bikeData.type === 'FOUND';
       return (
         <TouchableOpacity
-          onPress={() => {}}
+          onLongPress={() => {
```

Comment:
Some MDI knowledgable person would probably lecture you for this but I like it 👍 #HiddenFunctionality